proto/provenance/registry/v1/query.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+package provenance.registry.v1;
+
+option go_package          = "github.com/provenance-io/provenance/x/registry";
```

Comment:
issue: Let's put this stuff in a `types` directory (same with all the other registry proto files too).